i18n.json

```diff
@@ -0,0 +1,7 @@
+{
+  "locales": ["en", "es", "it"],
+  "defaultLocale": "en",
+  "pages": {
+    "*": ["common"]
+  }
+}
```

Comment:
We should add an empty line at the end of each file.

Reply:
done in commit https://github.com/capcatit/capcatit.github.io/pull/14/commits/0c434f527307558c2056b8e649f286d23177144d